src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets

```diff
@@ -41,6 +41,19 @@ It is shared between "legacy" binding projects and .NET 5 projects.
     </PropertyGroup>
   </Target>
 
+  <Target Name="_CollectGeneratedManagedBindingFiles">
+    <ItemGroup>
+      <_GeneratedManagedBindingFiles Include="$(IntermediateOutputPath)generated\*\*.cs" />
```

Comment:
Shouldn't this use `**\*.cs` instead of just `*\*.cs`?  It feels like this could make things more "future proof" if we ever add additional directories to the generated output.